extension/snippets/dvc-yaml.code-snippets

```diff
@@ -38,6 +38,19 @@
     ],
     "description": "Foreach stage template for dvc.yaml pipeline"
   },
+  "Matrix Pipeline Stage Template": {
+    "prefix": "dvc-pipeline-matrix-stage",
+    "body": [
```

Comment:
We don't have tests for these, right?

Reply:
No, we don't. As these shouldn't change often a demo should do.